ci/properties/matlab.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "MATLAB",
+    "description": "Build and test with MATLAB",
+    "iconName": "matlab",
+    "creator": "MathWorks",
+    "categories": [
+        "MATLAB",
```

Comment:
Probably should add Simulink here too.

Reply:
The [README](https://github.com/actions/starter-workflows#valid-properties) page said we can choose from languages present [here](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) only. Not sure if we can add categories on our own.